### DISCUSSION TITLE
Proper way to scale out/in ?

### DISCUSSION THREAD

**Discussion by aaronplata123** [minio/minio#14962] [Q&A] [answered]
Hello, awesome minio people! We are using helm (4.0.2) and tried scaling out by changing "pools: 1" to "pools: 2" and it worked like a charm! 
However, while trying a scaling in scenario (changing back to "pools: 1") some pods are stuck to "CrashLoopBackOff" status with message "ERROR Unable to initialize backend: pool(2nd) = http://minio-dev-{4...7}.minio-dev-svc.minio-dev.svc.cluster.local/export is not specified, please specify on server command line"
Is this the right way to scale out/in ? Thanks!

**Comment by harshavardhana** [comment, marked as answer]
> Hello, awesome minio people! We are using helm (4.0.2) and tried scaling out by changing "pools: 1" to "pools: 2" and it worked like a charm! However, while trying a scaling in scenario (changing back to "pools: 1") some pods are stuck to "CrashLoopBackOff" status with message "ERROR Unable to initialize backend: pool(2nd) = [http://minio-dev-{4...7}.minio-dev-svc.minio-dev.svc.cluster.local/export](http://minio-dev-%7B4...7%7D.minio-dev-svc.minio-dev.svc.cluster.local/export) is not specified, please specify on server command line" Is this the right way to scale out/in ? Thanks!

You can't scale back once you have scaled up. you need to decommission the scaled up setup first.

**Reply by aaronplata123** [reply]
How about the data? Can we re attach the warm storage for tiering of the new pool?

**Reply by klauspost** [reply]
Decommissioning moves the data from the decommissioning pool.
